src/index.js

```diff
@@ -108,7 +102,7 @@ export const createTimeago = (opts = {}) => {
       datetime() {
         this.convert()
       },
-      localeName() {
```

Comment:
let's keep the watcher for `localeName`, we need to re-render when `this.$timeago.locale` changes

Reply:
@egoist ah, I was not aware that also could update, then it should stay indeed.

Fixed.